src/AutoFilterer.Generators/FilterGenerator.cs

```diff
@@ -84,6 +84,10 @@ public partial class {className}Filter : PaginationFilterBase
                 if (TypeMapping.Mappings.TryGetValue(propertyType, out var mapped))
                     propertyType = mapped;
 
+                if (propertyType == "string")
```

Comment:
I'm not sure about it.
```suggestion
                if (propertyType.Equals(nameof(String), StringComparison.InvariantCultureIgnoreCase))
```
Might be better